languages/ocaml.scm

```diff
@@ -690,18 +690,20 @@
 (
   "with"
   .
-  "|" @singleline_delete
+  "|" @delete
   .
   (match_case)
+  (#single_line_only! )
```

Comment:
```suggestion
  (#single_line_only!)
```

Reply:
Unfortunately this is not a typo: see #383 
This is currently how predicates without parameters are formatted. If the space is removed, the tests fail because query file is no longer "correctly" formatted.